Sources/_CryptoExtras/RSA/RSA+BlindSigning.swift

```diff
@@ -227,6 +227,21 @@ extension _RSA.BlindSigning {
         public var publicKey: _RSA.BlindSigning.PublicKey<H> {
             _RSA.BlindSigning.PublicKey(self.backing.publicKey, self.parameters)
         }
+
+        public static func _createFromNumbers(n: some ContiguousBytes, e: some ContiguousBytes, d: some ContiguousBytes, parameters: Parameters) throws -> Self {
```

Comment:
Let's put a doc comment on here explaining what these do, along with the rest. This doc comment should make it clear that this constructor is strongly disfavoured as it's got extremely poor performance characteristics, and that it's provided as a convenience for when there is no better choice.